docs/api/target/class_balance.rst

```diff
@@ -33,11 +35,17 @@ The ``ClassBalance`` visualizer has a "compare" mode, where the train and test d
 
 .. code:: python
 
-    data = load_occupancy
+    from sklearn.model_selection import train_test_split
```

Comment:
Added import statements to the code block.

Reply:
good call here